flaml/automl.py

```diff
@@ -2502,7 +2517,11 @@ def cv_score_agg_func(val_loss_folds, log_metrics_folds):
             self._settings.get("retrain_full") if retrain_full is None else retrain_full
         )
         split_type = split_type or self._settings.get("split_type")
-        hpo_method = hpo_method or self._settings.get("hpo_method")
+        if lexico_objectives is None:
+            hpo_method = hpo_method or self._settings.get("hpo_method")
+        else:
+            hpo_method = "cfo"
```

Comment:
Need to output a warning if the hpo method is something else. Also, need to document this requirement in both docstr & website.

Reply:
address